src/VirtoCommerce.CatalogExportImportModule.Data/Services/CsvPagedPhysicalProductDataImporter.cs

```diff
@@ -0,0 +1,272 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using FluentValidation;
+using VirtoCommerce.CatalogExportImportModule.Core;
+using VirtoCommerce.CatalogExportImportModule.Core.Models;
+using VirtoCommerce.CatalogExportImportModule.Core.Services;
+using VirtoCommerce.CatalogModule.Core.Model;
+using VirtoCommerce.CatalogModule.Core.Model.Search;
+using VirtoCommerce.CatalogModule.Core.Search;
+using VirtoCommerce.CatalogModule.Core.Services;
+using VirtoCommerce.Platform.Core.Assets;
+using VirtoCommerce.Platform.Core.Common;
+
+namespace VirtoCommerce.CatalogExportImportModule.Data.Services
+{
+    public class CsvPagedPhysicalProductDataImporter : CsvPagedDataImporter<CsvPhysicalProduct>
+    {
+        private readonly IImportProductSearchService _importProductSearchService;
+        private readonly IImportCategorySearchService _importCategorySearchService;
+        private readonly IItemService _itemService;
+
+        public CsvPagedPhysicalProductDataImporter(
+            IImportPagedDataSourceFactory dataSourceFactory, IValidator<ImportRecord<CsvPhysicalProduct>[]> importRecordsValidator,
+            ICsvImportReporterFactory importReporterFactory, IBlobUrlResolver blobUrlResolver,
+            IImportProductSearchService importProductSearchService, IImportCategorySearchService importCategorySearchService,
+            IItemService itemService)
+            : base(dataSourceFactory, importRecordsValidator, importReporterFactory, blobUrlResolver)
+        {
+            _importProductSearchService = importProductSearchService;
+            _importCategorySearchService = importCategorySearchService;
+            _itemService = itemService;
+        }
+
+        public override string DataType { get; } = nameof(CatalogProduct);
```

Comment:
Use the module's constant DataTypes.PhysicalProduct instead nameof()

Reply:
Fixed